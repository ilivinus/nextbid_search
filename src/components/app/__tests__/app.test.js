import React from 'react';
import renderer from "react-test-renderer";
import App from '../index';

describe("#App",()=>{
    it("Should detect change in the App Component",()=>{
        let root = renderer.create(<App />);
        
        root = root.toJSON();
        
        expect(root).toMatchSnapshot();
    })
})
