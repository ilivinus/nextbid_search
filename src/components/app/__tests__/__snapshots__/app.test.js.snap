// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#App Should detect change in the App Component 1`] = `
<div
  className="App"
>
  <div
    className="search-container"
  >
    <p
      className="search-header"
    >
      <img
        alt="TheNextBid"
        className="logo"
        src="https://thenextbid.com/eeb9b1a373fc912868daaa578bb7ccc0.svg"
      />
      <strong>
        Search Engine
      </strong>
    </p>
    <form
      autoComplete="off"
    >
      <div
        className="suggestion"
      >
        <div
          className="suggestion"
        >
          <input
            className=""
            onChange={[Function]}
            onKeyDown={[Function]}
            placeholder="Search..."
            type="text"
            value=""
          />
          <button
            className="sb-button"
            onClick={[Function]}
            type="submit"
          >
            Submit
          </button>
        </div>
        <div
          className="suggestion-items"
        />
      </div>
    </form>
    <div />
  </div>
</div>
`;
